WORKSPACE

```diff
@@ -27,6 +27,16 @@ http_jar(
     url = "https://github.com/hsyed/bazel-deps/releases/download/v0.1.0/parseproject_deploy.jar",
 )
 
+http_archive(
```

Comment:
Is this to get for getting the tests running in docker ? rbe_ubuntu1604 , is that a "docker host" ?

Reply:
Yes, this is to get the config info necessary to run on rbe_ubuntu1604

Reply:
forgot to clarify, rbe_ubuntu1604 is an experimental hosted remote execution service that we are opening up for rule owners to try (will be sending out email to Bazel community in a few weeks, but I'm testing out the process with rules_kotlin atm - Bazel head as well as some other rules are running with this config as well). Once this is setup you might notice faster CI results from rbe_ubuntu1604 than from other configs. Having these rules be compatible with remote execution will open the door for remote execution bazel customers to run projects that depend on your rules with the experimental hosted remote build service.